co2calculator/distances.py

```diff
@@ -246,6 +263,8 @@ def geocoding_train_stations(loc_dict):
 
     :return: Name, country and coordinates of the found location
     """
+    station = TrainStation(**loc_dict)
+
     stations_df = pd.read_csv(
```

Comment:
Idea for future: Could we store all the data that is now in csv in an sqlite db? Then we wouldn't have to read the whole dataset everytime we need it but, we could just query the information we need through SQL queries

Reply:
good idea :)
For the stations csv-file, I'm not sure, because we obtain it from a subpackage, which is updated every now and then - but we could just update the sqlite db every time we release a new version? 
This would also be a good idea for storing ferry routes if we go for sth like suggested in #98

Reply:
Could be sth for the next Hackathon? :)

Reply:
Yes, sounds good! :)